### DISCUSSION TITLE
Proposed FRC standard for deal making

### DISCUSSION THREAD

**Discussion by aashidham** [filecoin-project/FIPs#721] [Filecoin Virtual Machine]
[FRC-NNNN: Deal making protocol between SPs and FVM smart contracts](https://github.com/filecoin-project/FIPs/blob/e913fea0f32cf412474f48600f29dbe9753b3d0c/FRCs/frc-nnnn-deal-making.md) is proposed in a [PR 693](https://github.com/filecoin-project/FIPs/pull/693). 

[Notion draft of this FRC is visible here](https://www.notion.so/pl-strflt/WIP-FRC-Deal-Client-Draft-4-v1-0-2f0fb97518994f97aa9ee397bbae33d3)

This post aims to open discussion around this proposed FRC. What key considerations are missing here? For the reviewers: what considerations need to be addressed before approval?

**Comment by anorth** [comment]
Is this essentially the same topic as https://github.com/filecoin-project/FIPs/discussions/604? What's the difference? Should one of these point to the other?

**Reply by aashidham** [reply]
That discussion was referencing an old standard. It was used to create v0.1-0.3 of that standard. The new standard incorporates SP frontrunning and 1-1 deal making and references a proposed PR.
